github-runner-provisioner/runner.go

```diff
@@ -5,6 +5,11 @@ import (
 	"github.com/aws/aws-sdk-go-v2/service/ec2"
 )
 
+var runners = map[string]func(context.Context, string, string, bool) error{
+	runnerLabel:            createMacM1Runner,
```

Comment:
should this be `macOSarm64` instead of `runnerLabel`?

Reply:
Ohh, I forgot to refactor the MacM1 object. Good catch!